packages/docs/components/link.md

```diff
@@ -62,6 +57,10 @@ Use an external link when:
   ```
 </figure>
 
+### Visited Links
+
+Odyssey has removed unique styling for `:visited` links. This is an intentional compromise that preferences user security and ease of maintenance over the `:visited` affordance.
```

Comment:
Do we need this section in docs? Seems like it might be better in changelog?

Reply:
Yeah, I was iffy on this. I do think it's better to document our intentional caveats up front, rather than make someone go digging.

@joejezowski-okta - flagging as something we may want to plan for in the new docs theme: rationale/notes that aren't as important as official guidance but may want in place to cover our butts.